Cilsil/Sil/Expressions/ExnExpression.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Cilsil.Sil.Expressions
+{
+    /// <summary>
+    /// Exception expression.
+    /// </summary>
+    [JsonObject]
+    public class ExnExpression : Expression
+    {
+        /// <summary>
+        /// The expression that has exception.
```

Comment:
can you make this a bit more descriptive? else give me a bit more explanation offline and i can help you find a better way to describe it. Also -- what modifications are you making to InferAnalyzeJson to translate this from the JSON?

Reply:
> can you make this a bit more descriptive? else give me a bit more explanation offline and i can help you find a better way to describe it. Also -- what modifications are you making to InferAnalyzeJson to translate this from the JSON?

Here is [link](https://github.com/facebook/infer/pull/1450) of companion PR in Infer backend side

Reply:
Update to  "An expression representing an exception variable".  Because as we see the changes in the other file, we create an `ExnExpression` with a `VarExpression`.